src/pkg/projects/Microsoft.NETCore.App/netcoreapp1.1.props

```diff
@@ -28,6 +28,12 @@
     <NETCoreApp11Dependency Include="NETStandard.Library">
```

Comment:
Sure.  I also had to lift some dependencies that were seemingly missed from 1.0/1.1 core.app packages.